CONTRIBUTING.md

```diff
@@ -58,13 +58,18 @@ Once you have babel running in watch mode, you can start making changes to parse
 
 If your pull request introduces a change that may affect the storage or retrieval of objects, you may want to make sure it plays nice with Postgres.
 
-* Run the tests against the postgres database with `PARSE_SERVER_TEST_DB=postgres npm test`. You'll need to have postgres running on your machine and setup [appropriately](https://github.com/parse-community/parse-server/blob/master/.travis.yml#L37) or use [`Docker`](#run-a-parse-postgres-with-docker).
+* Run the tests against the postgres database with `PARSE_SERVER_TEST_DB=postgres PARSE_SERVER_TEST_DATABASE_URI=postgres://postgres:password@localhost:5432/parse_server_postgres_adapter_test_database npm test`. You'll need to have postgres running on your machine and setup [appropriately](https://github.com/parse-community/parse-server/blob/master/.travis.yml#L43) or use [`Docker`](#run-a-parse-postgres-with-docker).
```

Comment:
Can you change the run test command to include `npm run testonly` so that it doesn't run the mongodb-runner

`
PARSE_SERVER_TEST_DB=postgres PARSE_SERVER_TEST_DATABASE_URI=postgres://postgres:password@localhost:5432/parse_server_postgres_adapter_test_database npm run testonly
`

Reply:
Done

Reply:
@cbaker6 Can you run the testonly locally? Making sure you don't have mongo running locally?

Reply:
I can try, I've always ran "npm test" because of the older versions of the docs

Reply:
`testonly` was added because we didn't want to keep starting the mongodb-runner. It makes testing faster.

Reply:
@dplewis I get 5 failures with "testonly" because of the missing mongo-runner. My guess is the failing tests are not properly labeled mongo only/dependent or have hard coded uri/references to adapters. This passes on travis because travis uses "npm test" and starts mongo-runner on the postgres test.

My proposal is that we switch back to "test" in contributions for this pull request and I can open up a new one later that fixes the mongo runner issue. To your point earlier, this should also save some time on the postgres test in travis.

Reply:
All of the errors say "MongoError:..."

Reply:
Sure I found the culprit, GridSFBucketStorageAdapter is hard coded to mong.

Let me see if this is a quick fix...

Reply:
I found it too just wanted you to see it.

Reply:
Is that the way to go? Should the grid... run on both mongo and postgres?

Reply:
No grid is MongoDB specific storage.

Reply:
Got you, I made the change

Reply:
Good to go

Reply:
In the future can you delete your fork, re create it and work off of branches instead of your master branch?

Reply:
@dplewis thanks, I didn't know this. I will try it next time

Reply:
Even though we squash commits. As you can see there are 60+ commits that carried over from previous PRs that you have done. A rule of thumb is to keep the master branch on your fork up to date and branch off of that before every PR.

Reply:
what i do is create a 'feature' branch that I put all my commits into and I periodically merge master into it.

making tons of small commits is a good practice imho (not disagreeing with dplewis, just adding my$.02).

Reply:
I have a develop branch that I use similar concept.